packages/default-theme/components/SwRegister.vue

```diff
@@ -145,15 +151,30 @@ export default {
   },
   setup() {
     const { login, register, loading, error } = useUser()
+    const countries = useCountries()
```

Comment:
```suggestion
    const { getCountries } = useCountries()
```
and same with salutations